digicampipe/visualization/time_resolution_plot.py

```diff
@@ -154,13 +157,13 @@ def plot_resol(data_file, legend, ax=None):
     mean_charge_all, std_charge_all, mean_t_all, std_t_all, true_pe = \
         load_data(data_file)
     if ax is None:
-        fig, ax = plt.subplots(1, 1, figsize=(8, 6), dpi=100)
+        fig, ax = plt.subplots(1, 1, figsize=(8, 8), dpi=100)
     # ax.plot([20*pde, 2e3*pde], [1, 1], 'r-', label='requirement B-TEL-1380')
     # ax.plot([20*pde, 20*pde], [.9, 1.1], 'r-', label=None)
     ax.plot([20*pde, 2e3*pde], [3, 3], 'b--', label='requirement B-TEL-1640')
-    ax.plot([20*pde, 20*pde], [1.4, 1.6], 'b-', label=None)
+    ax.plot([20*pde, 20*pde], [2.8, 3.2], 'b-', label=None)
```

Comment:
what is this ax.plot, plotting actually ? I am a bit confused

Reply:
it is plotting the small vertical line at the beginning of the lines for requirement (to indicate that the requirement stops there)